buddyboss/buddyboss-platform
# Improved reporting functionality

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2162 .

### How to test the changes in this Pull Request:

When activity component enabled/disabled, always report activity media instead of the main activity. Need to remove the dependency.
Account for Media threshold settings to auto-hide media instead of main activity threshold setting.

Make sure when Reporting for Activity Enabled then only reporting allowed for activity.
when Reporting for Media Enabled then only reporting allowed for media.

AND when activity all media hidden and if activity content is empty then only hide that Activity from the frontend.

### Proof Screenshots or Video



### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
